src/Modules/OAuth/Installer/OAuthInstaller.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace ForkCMS\Modules\OAuth\Installer;
+
+use ForkCMS\Modules\Extensions\Domain\Module\ModuleInstaller;
+use ForkCMS\Modules\Internationalisation\Domain\Translation\TranslationKey;
+use ForkCMS\Modules\OAuth\Backend\Actions\ModuleSettings;
+
+final class OAuthInstaller extends ModuleInstaller
+{
+    public function install(): void
+    {
+        $this->importTranslations(__DIR__ . '/../assets/installer/translations.xml');
+        $this->getOrCreateBackendNavigationItem(
+            TranslationKey::label('OAuth'),
+            ModuleSettings::getActionSlug(),
+            $this->getSettingsNavigationItem(),
+        );
+
+        $this->setSetting('client_id', null);
```

Comment:
Is dat nuttig om die by default op null op te slaan?

Reply:
fixed [abe9688](https://github.com/sumocoders/forkcms/pull/371/commits/abe96885c3e77320ace3ca5cd0e6f227571c921f)